Script/AtomicNET/AtomicNET/Core/Profiler.cs

```diff
@@ -12,12 +13,16 @@ public partial class Profiler : AObject
         {
 #if ATOMIC_PROFILING
             var profiler = AtomicNET.GetSubsystem<Profiler>();
-            profiler?.BeginBlock(name, file, line, color, (byte)status);
+            if (profiler != null)
```

Comment:
> We don't need AtomicNET here, AObject has GetSubsystem on it

👍 

> locally cache

You mean `Profiler.Block()` should not be static so that caller gets `Profiler` and reuses that reference?